### PR TITLE
GSoC 2021 Project Ideas

### PR DESCRIPTION
**Pull request by carolalynn22**
Signed-off-by: Carol Payne carolp@netflix.com

### CODE REVIEW

File: ASWF/gsoc_2021_README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+While OCIO is a color management library (written in C++ with a Python API), its only knowledge of color science comes from its execution of the transforms defined in the OCIO configuration file, written in YAML. These transforms are either defined by the end user in a custom OCIO config or inherited from the publicly available configs.  By specifying your desired config file in the local environment all OCIO compatible applications and software libraries will be able to see your defined color transform “universe”. You can then direct the transformation of image data from one defined OCIO.ColorSpace to another, in addition to the other transforms documented elsewhere - the options are endless!
```
Comment (doug-walker) [inline ASWF/gsoc_2021_README.md:10]: "OCIO.ColorSpace" --> "OCIO color space"

Minor: most of the .md files get wrapped to roughly 80 character lines.

File: ASWF/gsoc_2021_README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+involved with projects that will be used professionally on films!
+
+The mission of the ASWF is to increase the quality and quantity of
+contributions to the content creation industry's open source software base;
+to provide a neutral forum to coordinate cross-project efforts; to provide a
+common build and test infrastructure; and to provide individuals and
+organizations a clear path to participation in advancing our open source
+ecosystem.
+
+Member projects are focused primarily on visual effects and animation
```
Comment (doug-walker) [inline ASWF/gsoc_2021_README.md:30]: I would delete this paragraph and replace with something like:

In 2020, the ASWF projects OpenEXR, OpenTimelineIO, and OpenCue mentored GSoC students.  This year, OpenColorIO is the ASWF project that looks forward to welcoming and mentoring GSoC students.

File: ASWF/gsoc_2021_README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+## Application Instructions for Students
+
+Please review the CONTRIBUTING guidelines thoroughly before applying. OCIO maintains a contributor license agreement (CLA) which must be signed to accept contributions.
+
+**Successfully submitting a Pull Request to the OCIO project is a strict requirement prior to submitting an application.**
+This is not negotiable. We will not even read applications from students who have not submitted an acceptable PR to the project. It can be a truly minimal change -- for example, fixing a typo in an error message. You will find issues tagged "good first issue" on our GitHub repo, which are ideal candidates for your small trial PR to qualify for application.
+
+## Application template
+
+Please use this application template. Feel free to expand or tell us any other relevant details, but this is a good place to start. The applications will be judged by representatives of the OCIO technical advisory council.
```
Comment (doug-walker) [inline ASWF/gsoc_2021_README.md:47]: "technical advisory council" --> "technical steering committee"

File: ASWF/gsoc_2021_README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+* Student name:
+* School / year / area of study:
+* Relevant courses, materials studied, jobs, programming projects, skills, other open source projects you've contributed to. Go ahead and tell us anything about your background that you think we will find interesting for our projects.
+* Project proposal - please explain in detail and give specifics on acceptance criteria -- how we can measure your success, how we can test your code. Provide a weekly timeline for expected progress.
+* Please tell us which fix you contributed:
```
Comment (doug-walker) [inline ASWF/gsoc_2021_README.md:53]: "which fix you contributed:" --> "which test PR you had merged into to the OCIO repo:"

**Review by doug-walker — APPROVED** [review]

**Review by michdolan — APPROVED** [review]
Thanks again Carol! Once Doug's comments are addressed we can merge ASAP.

File: ASWF/gsoc_2021_README.md [hunk 5]
```diff
@@ -27,30 +47,38 @@ common build and test infrastructure; and to provide individuals and
...
-production, and at present include 5 distinct software packages. Student
-summer project applications can be for any of these packages individually,
-as well as potentially including tasks that bridge between them or provide
-common infrastructure (they share CI and other resources). The current
-member projects are:
-
-
+In 2020, the ASWF projects OpenEXR, OpenTimelineIO, and OpenCue mentored 
+GSoC students. This year, OpenColorIO is the ASWF project that looks forward 
+to welcoming and mentoring GSoC students.
```
Comment (doug-walker) [inline ASWF/gsoc_2021_README.md:52]: Minor: Add a blank line between this paragraph and the following heading.

**Review by doug-walker — APPROVED** [review]
Perfect, thank you Carol!

**Pull request merged**